musica/test/mechanism_configuration/test_arrhenius.py

```diff
@@ -0,0 +1,236 @@
+        name="stoich", 
+        reactants=[(2.0, A)], 
+        products=[(3.0, B), C],
+        gas_phase=gas
+    )
+    
+    reactants = arr2.reactants
+    assert len(reactants) == 1
+    assert isinstance(reactants[0], tuple)
+    assert reactants[0][0] == 2.0  # coefficient
+    assert reactants[0][1].name == "A"  # species
+    
+    products = arr2.products
+    assert len(products) == 2
+    # First product with coefficient
+    assert isinstance(products[0], tuple)
+    assert products[0][0] == 3.0
+    assert products[0][1].name == "B"
+    # Second product without coefficient
+    assert products[1].name == "C"
+    
+    print("✓ Reactants and products conversion working correctly")
+
+
+def test_arrhenius_serialize():
+    """Test the instance-based serialize method."""
+    A = mc.Species(name="A")
+    B = mc.Species(name="B")
+    gas = mc.Phase(name="gas", species=[A, B])
+    
+    arr = mc.Arrhenius(
+        name="serialize_test",
+        A=1.5e-11,
+        B=0.25,
+        C=75.0,
+        D=300.0,
+        E=2.0e-6,
+        reactants=[A],
+        products=[B],
+        gas_phase=gas,
+        other_properties={"custom_prop": "test_value"}
+    )
+    
+    # Test instance method (not static)
+    serialized = arr.serialize()
+    
+    assert isinstance(serialized, dict)
+    assert serialized["type"] == "ARRHENIUS"
+    assert serialized["name"] == "serialize_test"
+    assert serialized["A"] == 1.5e-11
+    assert serialized["B"] == 0.25
+    assert serialized["C"] == 75.0
+    assert serialized["D"] == 300.0
+    assert serialized["E"] == 2.0e-6
+    assert serialized["gas phase"] == "gas"
+    assert "reactants" in serialized
+    assert "products" in serialized
+    assert serialized["custom_prop"] == "test_value"
+    
+    print("✓ Instance-based serialize method working correctly")
+
+
+def test_arrhenius_serialize_static():
+    """Test the static serialize method for C++ object compatibility.""" 
+    # This test would ideally check the static method with a C++ object,
+    # but since we can't create _Arrhenius objects without the full build,
+    # we'll just verify the method exists and can be called
+    assert hasattr(mc.Arrhenius, 'serialize_static'), "Static serialize method should exist"
+    print("✓ Static serialize method exists for C++ compatibility")
+
+
+def test_arrhenius_gas_phase_types():
+    """Test gas_phase property with both Phase objects and strings."""
+    A = mc.Species(name="A")
+    B = mc.Species(name="B")
+    gas = mc.Phase(name="gas", species=[A, B])
+    
+    # Test with Phase object
+    arr1 = mc.Arrhenius(name="test1", gas_phase=gas)
+    assert arr1.gas_phase == "gas"
+    
+    # Test with string
+    arr2 = mc.Arrhenius(name="test2", gas_phase="liquid")
+    assert arr2.gas_phase == "liquid"
+    
+    # Test setting with Phase object
+    arr2.gas_phase = gas
+    assert arr2.gas_phase == "gas"
+    
+    print("✓ Gas phase type handling working correctly")
```

Comment:
Remove all prints like this throughout this test file. pytest should handle success/failure display